### PR TITLE
Disabled commenter image view on LoadThumbnail set to false

### PR DESCRIPTION
**Pull request by 4D17Y4**
#### What is it?
- [ ] Bug fix (user facing)
- [X] Feature (user facing)
- [ ] Code base improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- record videos
- create clones
- take over the world

#### Fixes the following issue(s)

- #4205 

#### Testing apk

[app-debug.zip](https://github.com/TeamNewPipe/NewPipe/files/5300447/app-debug.zip)

#### Agreement
- [X] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

**Review by Stypox — APPROVED** [review]
Thank you for squashing :-D
@TobiGr this can safely be merged inside 0.20.0 I think, as it does not introduce anything big or strange, and I tested

**Comment by Redirion** [comment]
but what about: 

itemBuilder.getImageLoader()
                .displayImage(item.getUploaderAvatarUrl(),
                        itemThumbnailView,
                        ImageDisplayConstants.DISPLAY_THUMBNAIL_OPTIONS);

this still causes the images to be fetched from the urls (causing network and device load unnecessarily).

**Comment by TobiGr** [comment]
@Redirion Can you link the code, please?

File: app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java [hunk 1]
```diff
@@ -85,11 +104,24 @@ public void updateFromItem(final InfoItem infoItem,
         }
         final CommentsInfoItem item = (CommentsInfoItem) infoItem;
 
+        preferences = PreferenceManager.getDefaultSharedPreferences(itemBuilder.getContext());
+
```
Comment (Redirion) [inline app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java:108]: @TobiGr it is directly in mid of the changes

Comment (Redirion) [reply app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java:108]: ah didn't get the correct lines, but just check the first changed file.

Comment (opusforlife2) [reply app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java:108]: The lines just below this one.

**Comment by 4D17Y4** [comment]
> this still causes the images to be fetched from the urls (causing network and device load unnecessarily).

https://github.com/TeamNewPipe/NewPipe/pull/4330#discussion_r496092124

@Redirion

**Review by Redirion — APPROVED** [review]

**Comment by Redirion** [comment]
its okay, no network traffic, but I wonder if loading a black thumbnail and then setting the view to GONE anyway is really needed. But this would just be a minor optimization. So from my side no obligations

File: app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java (hunk 1 above)
Comment (TobiGr) [reply app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java:108]: I went through the code and did not see it :D
Looks like I need a second pair of glasses...

**Pull request merged**